content/en/examples/controllers/job-pod-failure-policy-ignore.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: job-pod-failure-policy-ignore
+spec:
+  completions: 4
+  parallelism: 2
+  template:
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: main
+        image: docker.io/library/bash
+        command: ["bash"]
+        args:
+        - -c
+        - echo "Hello world! I'm going to exit with 0 (success)." && sleep 60 && exit 0
```

Comment:
Is 60 seconds enough for people to cordon the node?

Reply:
I think the main constraint is the pace of the user who goes through the steps - and it might get tricky for someone who goes through the example for the same time. Especially because there is an interim command to resolve the nodeName to issue. On the other hand, the same conclusion is true if someone evicts a pod from the second wave. Maybe 90s would be better.

Reply:
Maybe mention this time limit in the tutorial

Reply:
Added a caution note